code/game/objects/items/plushes.dm

```diff
@@ -705,7 +705,7 @@
 	name = "moth plushie"
 	desc = "A plushie depicting an adorable mothperson. It's a huggable bug!"
 	icon_state = "moffplush"
-	inhand_icon_state = null
```

Comment:
we dont seem to have the moffplush inhand state? if it was added to the plushies.dmi you will need to change the default in hand icon path to point to this

Reply:
The original intent was to just modify the base icon_state so that the plush would appear on or off regardless of whether it was in-hand or not. The modification of the inhand_icon_state was a holdover from the flashlight code (which has proper in-hand sprites)- I didn't actually make any in-hand sprites for the plushies, so I can set this back to null like the rest of the plushies and clean up the update_brightness proc to not modify it.

Reply:
please do as linters throw fits if there are missing in hand icons

Reply:
Updated now, got caught up in a few things last week.